### PR TITLE
[CHK-1383] feat(api): implement wallet creation endpoint

### PR DESCRIPTION
**Pull request by giovanniberti**
#### List of Changes

* implement endpoint to create wallets (both wallet persistence and call to payment gateway)
* add Docker compose files
* updated README

#### Motivation and Context

This PR adds the implementation for the creation endpoint to save wallets when invoked

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Depends on #8

### CODE REVIEW

**Review by pietro-tota — APPROVED** [review]
🚀

**Review by pietro-tota — APPROVED** [review]
🚀

**Comment by pietro-tota** [comment]
Add just some test for converters to increase code coverage

File: src/test/kotlin/it/pagopa/wallet/services/WalletServiceTest.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        given(npgClient.orderHpp(any(), any())).willReturn(Mono.just(WalletTestUtils.hppResponse()))
+
+        /* test */
+        val actual = walletService.createWallet()
+
+        /* assertions */
+        assertEquals(expected, actual)
+    }
+
+    @Test
```
Comment (pietro-tota) [inline src/test/kotlin/it/pagopa/wallet/services/WalletServiceTest.kt:39]: Add tests to cover missing returnUrl and securityToken in the received NPGResponse

**Review by pietro-tota — COMMENTED** [review]
Just a comment. Please fix also the SonarCloud detected code smells

File: src/test/kotlin/it/pagopa/wallet/services/WalletServiceTest.kt (hunk 1 above)
Comment (giovanniberti) [reply src/test/kotlin/it/pagopa/wallet/services/WalletServiceTest.kt:39]: Solved in [06962c6](https://github.com/pagopa/pagopa-wallet-service/pull/10/commits/06962c6879ab1fe84fffe90517ccc90bcd84dcc7)

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**